scp/lib/time_table.dart

```diff
@@ -4,6 +4,10 @@ import 'package:shared_preferences/shared_preferences.dart';
 import 'time_table_resources.dart';
 import 'package:url_launcher/url_launcher.dart';
 
+String theorySection = 'E';
```

Comment:
Why do we need these variables outside TimeTableState?

Reply:
I had changed it for accessing from another file. I guess it's not needed again so I will revert it back.